### DISCUSSION TITLE
Add a 'request' output port to the 'send request' block

### DISCUSSION THREAD

**Discussion by jfisher-staywell** [postmanlabs/postman-flows#293] [Ideas]
Hello,
I'm running some API requests via flows, and for one of my requests, I'm generating some information via a pre-request script.  That information is used in the body of the request.  However, that data is not part of the response.  It would be nice to be able to output the request to the console.
I realize that I can just look at the global postman console, but I'd like to be able to use the singular output console.

**Comment by saswatds** [comment]
@jfisher-staywell Thanks for the feedback. I will add this for consideration
